example/ck_tile/01_fmha/fmha_fwd.cpp

```diff
@@ -175,61 +176,49 @@ auto get_elimit<ck_tile::fp8_t>(std::string init_method)
     }
 }
 
-int num_splits_heuristic(int batch_nhead_mblocks, int num_SMs, int num_n_blocks, int max_splits)
+int num_splits_heuristic(int batch_nhead_mblocks, int num_SMs, int max_splits)
```

Comment:
Is it possible to move this function into fmha_fwd.hpp? so that 3rd party could call it directly